<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsCategorized3DRendererWidget</class>
 <widget class="QWidget" name="QgsCategorized3DRendererWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>368</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Value</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QgsFieldExpressionWidget" name="mExpressionWidget" native="true">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="symbolLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Symbol</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="Qgs3DSymbolButton" name="mBtnChangeCategorizedSymbol">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_10">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Color ramp</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QgsColorRampButton" name="mBtnColorRamp">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QTreeView" name="mViewCategories">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::InternalMove</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::IgnoreAction</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="mBtnAddCategories">
       <property name="text">
        <string>Classify</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mBtnAddCategory">
       <property name="toolTip">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="../../images/images.qrc">
         <normaloff>:/images/themes/default/symbologyAdd.svg</normaloff>:/images/themes/default/symbologyAdd.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mBtnDeleteCategories">
       <property name="toolTip">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset resource="../../images/images.qrc">
         <normaloff>:/images/themes/default/symbologyRemove.svg</normaloff>:/images/themes/default/symbologyRemove.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mBtnDeleteAllCategories">
       <property name="text">
        <string>Delete All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mBtnDeleteUnusedCategories">
       <property name="text">
        <string>Delete Unused</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>mViewCategories</zorder>
  <zorder>mBtnChangeCategorizedSymbol</zorder>
  <zorder>symbolLabel</zorder>
  <zorder>label_10</zorder>
  <zorder>mExpressionWidget</zorder>
  <zorder>label_3</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFieldExpressionWidget</class>
   <extends>QWidget</extends>
   <header location="global">qgsfieldexpressionwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsColorRampButton</class>
   <extends>QToolButton</extends>
   <header>qgscolorrampbutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsSymbolButton</class>
   <extends>QToolButton</extends>
   <header>qgssymbolbutton.h</header>
  </customwidget>
  <customwidget>
   <class>Qgs3DSymbolButton</class>
   <extends>QWidget</extends>
   <header location="global">qgs3dsymbolbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mExpressionWidget</tabstop>
  <tabstop>mBtnChangeCategorizedSymbol</tabstop>
  <tabstop>mBtnColorRamp</tabstop>
  <tabstop>mViewCategories</tabstop>
  <tabstop>mBtnAddCategories</tabstop>
  <tabstop>mBtnAddCategory</tabstop>
  <tabstop>mBtnDeleteCategories</tabstop>
  <tabstop>mBtnDeleteAllCategories</tabstop>
 </tabstops>
 <resources>
  <include location="../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
